### ISSUE TITLE
Provide more instructions for developing and testing locally

### ISSUE THREAD

**Issue by cmatskas** [DarqueWarrior/generator-team#10]
Hi @DarqueWarrior,
I love contributing to open source projects and this one is a great example of how we can make better tools for us and our users. I understand that this is a pet project that has now gained a lot of traction and interest by the community so I would like to suggest that we make it easier for people to contribute. 

It would be great if we could add a Contributing.MD or expand the existing sections on the `Readme.MD` to explain how to download, develop and run on the local machine. This would help with people that don't have good experience working with local `npm` packages for example (like me...)

Thoughts?

**Comment by cmatskas** [comment]
I also tried to run the tests against my branch. It appears that the tests need a local instance of TFS running. We may want to make this configurable to allow devs to point to a VSTS instance to run the tests accordingly. It's a bit unreasonable and potentially not always applicable (developing on my Mac as we speak) to demand for devs to have their own local TFS instance. Just my $0.02

**Comment by DarqueWarrior** [comment]
I will spin up a vm and clone fresh to make sure the test work.

**Comment by cmatskas** [comment]
Embarrassing.... Thanks Donovan! However, we still have a dependency on a local TFS to be able to run the tests, right?

**Comment by DarqueWarrior** [comment]
Only the integration tests and that is controlled via Environment Vars.  There is a readme.md in the Integration folder that explains the required Environments Vars to set to run those tests. The unit tests have no dependencies. I just spun up a Server 2016 in Azure. I installed Git, Node 6.11 and upgraded the NPM installed with Node. I then cloned the repo, linked it as described in the Wiki then ran npm test and all unit tests passed.
